hw/ip/usbdev/dv/README.md

```diff
@@ -20,20 +19,26 @@ For detailed information on USBDEV design features, please see the [USBDEV HWIP
 ## Testbench architecture
 The USBDEV testbench is based on the [CIP testbench architecture](../../../dv/sv/cip_lib/README.md).
 
+## Block diagram
+![Block diagram](./doc/tb.svg)
+
 ### Top level testbench
-The top-level testbench is located at `hw/ip/usbdev/dv/tb/tb.sv`.
-It instantiates the USBDEV DUT module `hw/ip/usbdev/rtl/usbdev.sv`.
+The top-level testbench is located at `hw/ip/usbdev/dv/tb/tb.sv`. It instantiates the USBDEV DUT module `hw/ip/usbdev/rtl/usbdev.dv` along with the following support modules:
+
+- `hw/ip/usbdev/rtl/usbdev_aon_wake` which provides monitoring of the USB in low power modes, and produces wake events.
+- `hw/ip/prim_generic/rtl/prim_generic_usb_diff_rx` which models an external differential receiver, required for proper compliance with the USB 2.0 Protocol Specification.
 
-USBDEV has the following interfaces, which the testbench instantiates and connects and registers with `uvm_config_db`:
+The following interfaces are also used, which the testbench instantiates, connects and registers with `uvm_config_db`:
 - [Clock and reset interfaces](../../../dv/sv/common_ifs/README.md) for the USB and AON clock domains.
 - A [TileLink interface](../../../dv/sv/tl_agent/README.md).
   USBDEV is a TL-UL device, which expects to communicate with a TL-UL host.
   In the OpenTitan SoC, this will be the Ibex core.
-- A `usb20_block_if` representing the actual USB interface.
-  This is using a class that has been developed in parallel with `usb20_if`.
-  Eventually, they will hopefully be merged again.
+- An interface `usb20_if` that provides the ASIC-level interface to the USB DPI model, using true USB signaling with tri-stating, pull ups and no externally-visible driver enable signals.
+- An interface `usb20_block_if` representing the USB interface between the `usb20_agent` and the DUT. This interface carries additional signals that are not part of the USB. The additional both permit the test of all bus configurations and presently aid the operation of the usb20_monitor which requires knowledge of whether the USB is being driven by the agent or the DUT.
```

Comment:
I think you want "The additionally both permit" here? Also, don't you want one sentence per line? I think there are some missing newlines here.